src/main/java/com/feedzai/commons/sql/abstraction/dml/dialect/SqlBuilder.java

```diff
@@ -519,6 +519,30 @@ public static View createView(final String name) {
         return new View(name);
     }
 
+    /**
+     * Creates a case expression.
+     *
+     * @param condition The name of the view.
+     * @param trueAction The name of the view.
+     * @return The case when representation.
+     */
+    public static Case caseWhen(final Expression condition, final Expression trueAction) {
+        return Case.caseWhen(condition, trueAction);
+    }
+
+    /**
+     * Creates a case when else expression.
+     *
+     * @param condition The name of the view.
+     * @param trueAction The name of the view.
+     * @param falseAction The name of the view.
+     * @return The case when representation.
+     */
+    public static CaseElse caseElse(final Expression condition, final Expression trueAction,
```

Comment:
Again, I believe we should not have this, where else is only accessible through the above expression.